media/public/include/MediaCommon.h

```diff
@@ -59,6 +59,9 @@ enum class MediaSourceType
     VIDEO
 };
 
+/**
+ * @brief The supported types of config types.
```

Comment:
Improve the wording, something like "The supported media source types".

Not related to your change but should we rename this "MediaSourceType"